x-pack/test/functional_with_es_ssl/apps/cases/group1/view_case.ts

```diff
@@ -1013,11 +1013,22 @@ export default ({ getPageObject, getService }: FtrProviderContext) => {
     describe('Tabs', () => {
       createOneCaseBeforeDeleteAllAfter(getPageObject, getService);
 
+      it('renders tabs correctly', async () => {
+        await testSubjects.existOrFail('case-view-tab-title-activity');
+        await testSubjects.existOrFail('case-view-tab-title-files');
+        // there are no alerts in stack management yet
+      });
+
       it('shows the "activity" tab by default', async () => {
         await testSubjects.existOrFail('case-view-tab-title-activity');
         await testSubjects.existOrFail('case-view-tab-content-activity');
       });
 
+      it("shows the 'activity' tab when clicked", async () => {
```

Comment:
Does this test make sense here following the previous one?

We are already in the activity tab, I don't know how relevant it is to check that nothing changes when we click it.

Maybe we could go to files and back instead?